priv/companies/uplearn.exs

```diff
@@ -0,0 +1,20 @@
+# Company file for Up Learn
+# Created on: 2024-10-30
+
+%{
+  name: "Up Learn",
+  website: "https://uplearn.co.uk/",
+  github: "https://gitlab.com/up-learn-uk",
+  # reference lib/companies/industries.ex for a list of recommended industries to use here
+  industry: "Education",
```

Comment:
@wladimirgramacho `Education` isn't part of the GICS industries. Can you please update to one listed on https://www.msci.com/our-solutions/indexes/gics ? My suggestion would be `Information Technology`

Reply:
```suggestion
  industry: "Information Technology",
```